packages/synapse-react-client/src/components/TimelinePlot/phasesQueryResponseData.ts

```diff
@@ -0,0 +1,181 @@
+import { QueryResultBundle } from '@sage-bionetworks/synapse-types'
```

Comment:
Hard coded phase data from a table that Christina created (syn52408661).  We don't really need the observationTimeMax or observationTimeUnits columns (these are just estimates for the max time in a phase, and they want the x-axis range to be based on the observational data instead).

Reply:
Are all aware that this is hard-coded (so changes they make to the table won't affect the code unless we update this entire object)?

What's the motivation for hard-coding this?